types/types.d.ts

```diff
@@ -12,7 +12,7 @@ declare module "fastify" {
     destroySession(callback: (err?: Error) => void): void;
   }
 
-  interface Session extends Record<string, any> {
+  interface Session {
```

Comment:
So this forces user to declare their own session struct like: 
```ts
declare module 'fastify' {
  interface Session {
    id?: number;
    foo: string;
  }
}

```

Reply:
yep 🙂

Reply:
Nice. Can you just add the `id?: number` property in the example in the readme?

Reply:
[`89c375b` (#14)](https://github.com/fastify/session/pull/14/commits/89c375bc9df3d758728cbb3b870c117ee0c566af)?

Reply:
Yes